hypothesis-python/scripts/basic-test.sh

```diff
@@ -38,31 +40,24 @@ if [ "$(python -c 'import sys; print(sys.version_info[:2] in ((2, 7), (3, 6)))')
   exit 0
 fi
 
-for f in tests/nocover/test_*.py; do
-  $PYTEST "$f"
-done
+$PYTEST tests/nocover/
 
 # fake-factory doesn't have a correct universal wheel
 pip install --no-binary :all: faker
 $PYTEST tests/fakefactory/
 pip uninstall -y faker
 
 if [ "$(python -c 'import platform; print(platform.python_implementation())')" != "PyPy" ]; then
-  if [ "$(python -c 'import sys; print(sys.version_info[0] == 2 or sys.version_info[:2] >= (3, 4))')" == "True" ] ; then
```

Comment:
Double checking: this is fine to remove because we never test with anything less than 2.7 or 3.4?

Reply:
Correct.  It has had no effect since we dropped support (and CI) for Python 3.3